third_party/skia/include/core/SkStream.h

```diff
@@ -317,7 +317,7 @@ class SK_API SkFILEStream : public SkStreamAsset {
      *  beginning of the SkFILEStream and size bytes later will be the end.
      *  The C FILE stream will be closed in the destructor.
      */
-    explicit SkFILEStream(FILE* file, size_t size);
+    explicit SkFILEStream(SkFile* file, size_t size);
```

Comment:
I'm unsure if those are our modifications from upstream or not, as it's hard to read ( e.g. whats applying a new change and what's undoing a change ). 

But if this is our change, we could consider sending this upstream with the justification for better portability.

No need for action on this PR.

Reply:
It's our change for Starboard, not sure this is a good candidate for sending upstream.
`#if defined(STARBOARD)
#include "starboard/file.h"
#define SkFile SbFilePrivate
#else
#include <stdio.h>
#define SkFile FILE
#endif`